src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/InitializeStaticFieldsInlineTests.cs

```diff
@@ -169,6 +169,7 @@ class C
     static C()
     {
         Console.CancelKeyPress += (o, e) => s = string.Empty;
+        Console.CancelKeyPress -= (o, e) => s = string.Empty;
```

Comment:
💡 I see tests for most lambda forms, but would like to see one for anonymous function syntax:

```csharp
delegate { s = string.Empty; }
```
```

Reply:
What's the equivalent for vbnet?

Reply:
💭 I don't think there is one

Reply:
Ok! Thanks!